ports/mimxrt/pin.c

```diff
@@ -171,7 +171,7 @@ const machine_pin_obj_t *pin_find(mp_obj_t user_obj) {
         return pin_obj;
     }
 
-    mp_raise_ValueError(MP_ERROR_TEXT("Pin doesn't exist"));
+    mp_raise_ValueError(MP_ERROR_TEXT("Pin does not exist"));
```

Comment:
I reverted this change, it should use contracted form to match everywhere else.

Reply:
The intent was to be more easily understood by users who have English as a 2nd language.   I see the point about consistency.

Reply:
There was a commit/PR from a while ago that went through and contracted everything, with the main aim to reduce code size.